kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/installation/kroxylicious/Kroxylicious.java

```diff
@@ -82,6 +88,50 @@ public void deployPortIdentifiesNodeWithNoFilters(String clusterName) {
                         clusterName, Constants.KROXYLICIOUS_INGRESS_CLUSTER_IP));
     }
 
+    /**
+     * Deploy port identifies node with tls and no filters.
+     *
+     * @param clusterName the cluster name
+     */
+    public void deployPortIdentifiesNodeWithTlsAndNoFilters(String clusterName) {
+        createCertificateConfigMap(deploymentNamespace);
+        resourceManager.createResourceFromBuilder(
+                KroxyliciousKafkaProxyTemplates.defaultKafkaProxyDeployment(deploymentNamespace, Constants.KROXYLICIOUS_PROXY_SIMPLE_NAME),
```

Comment:
naming -  the thing you are create is not a deployment.  It is the resource or proxy CR.